### PR TITLE
amam/fix translation staging

### PR DESCRIPTION
**Pull request by mustofa-binary**
# Description

Fix all text translation crowdin

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

**Comment by ashkanx** [comment]
#### 📝Changed routes:


| [**/about/_leaders**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_leaders) | [**/about/_our-history**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_our-history) |
|:-:|:-:|
|<a href="https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_leaders">Screenshot of /about/_leaders</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/De9T4kqgj7W9HNjujNntUG3iUqw_.png">(view full size)</a> |<a href="https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_our-history">Screenshot of /about/_our-history</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/2qa2SqAJET5L_1_EMeXFlXKmVik_.png">(view full size)</a> |



| [**/about/_our-numbers**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_our-numbers) | [**/about/_story_constant**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_story_constant) |
|:-:|:-:|
|<a href="https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_our-numbers">Screenshot of /about/_our-numbers</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/zEPrv3Rix_SAJeYt8PlNcsTI000_.png">(view full size)</a> |<a href="https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/about/_story_constant">Screenshot of /about/_story_constant</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/oIA00PeDNDCsb62Nvu7WtfXhYi0_.png">(view full size)</a> |



| [**/dbot**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/dbot) | [**/dmt5**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/dmt5) |
|:-:|:-:|
|<a href="https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/dbot">Screenshot of /dbot</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/NYbTqOaPAzvPG0yHO6pIbDbF3Co_.png">(view full size)</a> |<a href="https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/dmt5">Screenshot of /dmt5</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Sha1fFW1qd1gABP2rB0rOFpVjV0_.png">(view full size)</a> |



And 25 other routes:
- [**/dtrader**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/dtrader)
- [**/help-centre/_search-results**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/help-centre/_search-results)
- [**/help-centre/account**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/help-centre/account)
- [**/help-centre/deposits-and-withdrawals**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/help-centre/deposits-and-withdrawals)
- [**/help-centre**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/help-centre)
- [**/help-centre/security**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/help-centre/security)
- [**/help-centre/trading**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/help-centre/trading)
- [**/home/_what-our-clients-say**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/home/_what-our-clients-say)
- [**/markets/_available-options**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/markets/_available-options)
- [**/markets/_available-trades**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/markets/_available-trades)
- [**/markets/_other-markets**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/markets/_other-markets)
- [**/markets/_what-trade**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/markets/_what-trade)
- [**/markets/_why-trade**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/markets/_why-trade)
- [**/markets/forex/_forex**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/markets/forex/_forex)
- [**/partners/_about-deriv**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/_about-deriv)
- [**/partners/_deriv-numbers**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/_deriv-numbers)
- [**/partners/_hero**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/_hero)
- [**/partners/_why-choose-us**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/_why-choose-us)
- [**/partners/affiliate-ib/_deriv-ib-programme**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/affiliate-ib/_deriv-ib-programme)
- [**/partners/affiliate-ib/_faq-data**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/affiliate-ib/_faq-data)
- [**/partners/affiliate-ib/_faq**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/affiliate-ib/_faq)
- [**/partners/affiliate-ib/_why-us**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/affiliate-ib/_why-us)
- [**/partners/payment-agent/_faq-data**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/payment-agent/_faq-data)
- [**/partners/payment-agent/_faq**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/partners/payment-agent/_faq)
- [**/payment-methods/_expanded-list**](https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx/payment-methods/_expanded-list)



Commit 2eef7b5f0fd5ec217f9f173adfa186e759fb18c5 (https://deriv-com-git-fork-mustofa-binary-amam-fixtranslationstaging.binary.sx).

File: src/components/custom/_dhow-it-works.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const DHowItWorks = ({ Video, title }) => {
                     {localize('How it works')}
                 </StyledHeader>
                 <StyledHeader as="h2" align="center" size="var(--text-size-header-1)" mt="0.8rem">
-                    {title}
+                    {localize(title)}
```
Comment (arash-fs) [inline src/components/custom/_dhow-it-works.js:25]: can we pass `localize` from the parent? :thinking:

File: src/components/custom/_dnumbers.js [hunk 2]
```diff
@@ -31,9 +32,9 @@ const DNumbers = ({ items, justify }) => {
                 {items.map((item, index) => (
                     <NumberWrapper key={index}>
                         <Header as="h2" align="center">
-                            {item.title}
+                            {localize(item.title)}
```
Comment (arash-fs) [inline src/components/custom/_dnumbers.js:35]: Also here

File: src/components/custom/_dnumbers.js [hunk 3]
```diff
@@ -31,9 +32,9 @@ const DNumbers = ({ items, justify }) => {
...
                         </Header>
-                        <Text align="center">{item.subtitle}</Text>
+                        <Text align="center">{localize(item.subtitle)}</Text>
```
Comment (arash-fs) [inline src/components/custom/_dnumbers.js:37]: Also here, I think it's better to pass `localize` from up to down.

File: src/components/elements/card.js [hunk 4]
```diff
@@ -322,10 +322,10 @@ export const NavCard = ({ icon: Icon, title, content, to, style, external, targe
                 <Icon />
                 <NavContent>
                     <ResponsiveHeader size="var(--text-size-xs)" lh="1.14" mb="0.8rem">
-                        {title}
+                        {localize(title)}
                     </ResponsiveHeader>
                     <ResponsiveText size="var(--text-size-xxs)" color="grey-5">
-                        {content}
+                        {localize(content)}
```
Comment (arash-fs) [inline src/components/elements/card.js:328]: and here

File: src/pages/about/_leaders.js [hunk 5]
```diff
@@ -299,14 +299,14 @@ const Leaders = () => {
                                     <QueryImage data={data[leader.image]} alt={leader.name} />
                                     <DescriptionWrapper>
                                         <Text lh="1.71" size="var(--text-size-xs)" color="white">
-                                            {leader.description}
+                                            {localize(leader.description)}
```
Comment (arash-fs) [inline src/pages/about/_leaders.js:302]: and here ...

File: src/pages/about/_leaders.js [hunk 6]
```diff
@@ -299,14 +299,14 @@ const Leaders = () => {
...
                                         </Text>
                                     </DescriptionWrapper>
                                 </Box>
                                 <StyledHeader as="h4" align="center" lh="3.6rem">
                                     {leader.name}
                                 </StyledHeader>
-                                <Text align="center">{leader.position}</Text>
+                                <Text align="center">{localize(leader.position)}</Text>
```
Comment (arash-fs) [inline src/pages/about/_leaders.js:309]: and here ...

File: src/components/custom/_dhow-it-works.js (hunk 1 above)
Comment (mustofa-binary) [reply src/components/custom/_dhow-it-works.js:25]: decided to use Localize instead 🙏

File: src/pages/home/_what-our-clients-say.js [hunk 7]
```diff
@@ -179,33 +178,24 @@ ClientSlide.propTypes = {
...
-    img_path: PaulImage,
-    quote: localize(
-        'Customer service support very awesome and first to respond to queries and helping on marketing part. No much struggle introducing new members to binary.com since the company name already have a known truck of good ethics.',
+    quote: (
+        <Localize translate_text="It surpassed my expectations. Binary got it right with Deriv. Trading on the platform is excellent and it allows for making accurate graphical analyses of the market and adding support and resistance markings with the use of horizontal lines, RSI, FIBO and much more." />
     ),
     index: 2,
 }
 
-const our_client_slides = [roberto, fabio, paul]
```
Comment (arash-fs) [inline src/pages/home/_what-our-clients-say.js:208]: bye, paul :smile:

Comment (mustofa-binary) [reply src/pages/home/_what-our-clients-say.js:208]: 😂

**Pull request merged**